.circleci/config.yml

```diff
@@ -178,7 +178,34 @@ jobs:
             --dotenv-path $(pwd)/.env \
             --justfile ../../../single.just \
             simulate
-  
+  just_simulate_eth_017-presigned-pause:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: just simulate mainnet 017-presigned-pause
+          command: |
+            just install
+            cd tasks/eth/017-presigned-pause
+            just \
+              --dotenv-path .env \
+              --justfile ../../../presigned-pause.just \
+              install
+            forge build
+            export SIMULATE_WITHOUT_LEDGER=1
+            export SCRIPT=PresignPauseFromJson_eth_017
+            export FOUNDRY_SENDER=0x3041BA32f451F5850c147805F5521AC206421623
+            export TEST_SENDER=0x3041BA32f451F5850c147805F5521AC206421623
```

Comment:
Where is this address from?

Reply:
Looks like presigner uses it? 

https://github.com/ethereum-optimism/superchain-ops/blob/maur/add-ci/presigned-pause.just#L58-L66

I just grabbed this from the last PSP CI job.